Config netty param in HttpServer

liutang123 opened the pull request:
for #4224
Add three http param to control the netty behavior.

liutang123 committed:
Config netty param in HttpServer

for #4224
Add three http param to control the netty behavior.

kangkaisen reviewed (dismissed):
+1

kangkaisen commented:
@liutang123 Hi. please add these new configs to doc.

morningman commented on fe/fe-core/src/main/java/org/apache/doris/http/HttpServer.java:
```diff
@@ -17,6 +17,17 @@
 
 package org.apache.doris.http;
 
+import io.netty.bootstrap.ServerBootstrap;
```
remain the import order.

imay replied:
We'd better to provide some config for popular IDE

liutang123 committed:
Add to docs

Add  http_max_line_length and http_max_header_size to docs.
Because `maxChunkSize` is removed after Netty 5.X, we don't add http_max_chunk_size to docs.

liutang123 committed:
Remain import order

liutang123 commented:
> @liutang123 Hi. please add these new configs to doc.

Because `maxChunkSize` is removed after netty 5.x, so I don't add it to docs and just add the other two.

imay reviewed (approved):
LGTM
